content/sales.rst

```diff
@@ -0,0 +1,152 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes operational Sales details.
+
+
+Product Demo
+============
+
+Please see the `DemoSession` wiki page for the demo session structure.
+
+
+Demo text template to use
+-------------------------
+
+The demo session content is similar to the documentation in that it is aimed at
+system administrators and other individuals involved.
+
+Please send your criteria of requirements so that we can
+better answer your questions in advance of the demo.
+
+For those involved in procurement, please contact the Sales team.
+
+**Pre-demo session:**
+
+We recommend obtaining the trial version and installing until the
+`initial stage <https://www.sftpplus.com/documentation/sftpplus/latest/getting-started.html>`_
+
+In that way, you are able to continue after the demo session.
+
+
+Items to consider agreeing to a demo:
+-------------------------------------
+
+Is the customer actively pursuing a trial or considering a trial?
+
+If they are still considering a trial, it would be best to get a list of
+requirements first, and send through response.
+
+
+Is it attended by administrators / individuals involved in managing?
+--------------------------------------------------------------------
+
+Like readership of the documentation page, the session should be attended by
+at least an administrator or individual involved in managing the roll-out.
```

Comment:
what should we do if this is not true and the demo is attended by a sales rep or a support person?

Reply:
I think this section can be cut out since the 'who' would be the one sending through the requirements anyway.
I don't think there has been a case where the demo is being attended by the wrong people